components/NavbarV2.js

```diff
@@ -0,0 +1,206 @@
+                  </NavItem>
+                </Link>
+                <Link href="/become-a-host">
+                  <NavItem as={Container} mt={16} mb={16}>
+                    <FormattedMessage id="pricing.fiscalHost" defaultMessage="For Fiscal Hosts" />
+                  </NavItem>
+                </Link>
+              </NavLinkContainer>
+            </PopupMenu>
+
+            <PopupMenu
+              Button={({ onClick }) => (
+                <NavButton as={StyledLink} onClick={onClick} whiteSpace="nowrap">
+                  <FormattedMessage id="ContributionType.Product" defaultMessage="Product" />
+                  <ChevronDown size={20} />
+                </NavButton>
+              )}
+              placement="bottom-start"
+            >
+              <NavLinkContainer>
+                <Link href="/pricing">
+                  <NavItem as={Container} mt={16} mb={16}>
+                    <FormattedMessage id="menu.pricing" defaultMessage="Pricing" />
+                  </NavItem>
+                </Link>
+                <Link href="/how-it-works">
+                  <NavItem as={Container}>
+                    <FormattedMessage id="menu.howItWorks" defaultMessage="How it Works" />
+                  </NavItem>
+                </Link>
+                <Link href="/fiscal-hosting">
+                  <NavItem as={Container} mt={16} mb={16}>
+                    <FormattedMessage id="editCollective.fiscalHosting" defaultMessage="Fiscal Hosting" />
+                  </NavItem>
+                </Link>
+              </NavLinkContainer>
+            </PopupMenu>
+
+            <PopupMenu
+              Button={({ onClick }) => (
+                <NavButton as={StyledLink} onClick={onClick} whiteSpace="nowrap">
+                  <FormattedMessage id="company" defaultMessage="Company" />
+                  <ChevronDown size={20} />
+                </NavButton>
+              )}
+              placement="bottom-start"
+            >
+              <NavLinkContainer>
+                <Link href="/e2c">
+                  <NavItem as={Container} mt={16} mb={16}>
+                    <Span>E2C</Span>
+                  </NavItem>
+                </Link>
+                <a href="https://blog.opencollective.com/">
+                  <NavItem as={Container} mb={16}>
+                    <FormattedMessage id="company.blog" defaultMessage="Blog" />
+                  </NavItem>
+                </a>
+              </NavLinkContainer>
+            </PopupMenu>
+            <Link href="/help">
+              <NavButton as={Container} whiteSpace="nowrap">
+                <FormattedMessage defaultMessage="Help & Support" />
+              </NavButton>
+            </Link>
+            <Container borderRight="2px solid #DCDDE0" height="20px" padding="5px" />
+          </NavList>
+        </Hide>
+        <NavButton as={StyledLink} href="/search?type=COLLECTIVE">
```

Comment:
For now I am directing the user to `/search?type=COLLECTIVE` due to, https://github.com/opencollective/opencollective/issues/5454. I've also opened a new issue to suggest making this the default behavior on mobile. 🤔 

https://github.com/opencollective/opencollective/issues/5532